apps/controllo-ore-x/src/app/_shared/classes/upsert-form-helper.class.ts

```diff
@@ -107,9 +107,9 @@ export abstract class UpsertFormHelper<
       }
 
       this._dataService.getOne(this.entityId).subscribe({
-        next: (res) => {
-          this.patchForm(res?.data);
-          this.currentEntity = res?.data;
+        next: (res: any) => {
```

Comment:
```suggestion
        next: (res: ReadT) => {
```
You should be able to use the generics passed to the class I think